app/components/donation/project-header.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+const {
+  Component
+} = Ember;
+
+export default Component.extend({
+  classNames: ['project-header']
```

Comment:
This is from the old code and styles, unchanged, just copied to a new location. I agree, but it should be part of a "fix" issue. I think the styles are from before the switch BEM, even.